src/app/restapi.lua

```diff
@@ -1,232 +1,160 @@
-local restapi = {
-    incubator = nil
-}
+local restapi = {incubator = nil}
 
 -------------------------------------
--- ! @function max_temp   print the current temperature
+-- ! @function change config   modify the current config.json file
 --
--- !	@param req  				server request
+-- !	@param req  		      		server request
 -------------------------------------
-function restapi.max_temp_get(req)
 
-    local body_data = {
-        message = "success",
-        maxtemp = restapi.incubator.max_temp
-    }
-
-    local body_json = sjson.encode(body_data)
+function restapi.read_config()
+    -- open file with read permission
+    local file = io.open("config.json", "r")
 
-    return {
-        status = "200 OK",
-        type = "application/json",
-        body = body_json
-    }
+    if file ~= nil then
+        -- "*a" read all the file
+        local config_json = file:read("*a")
+        -- then close the file
+        file:close()
 
-end -- end function
+        local config_data = sjson.decode(config_json)
+        return config_data
+    else
+        print("[!] Failed to read JSON file")
+        return nil
+    end -- if end
+end -- function end
 
 -------------------------------------
--- ! @function min_temp   print the current temperature
+-- ! @function change config   modify the current config.json file
 --
--- !	@param req  				server request
+-- !	@param req  		      		server request
 -------------------------------------
-function restapi.min_temp_get(req)
-    local body_data = {
-        message = "success",
-        mintemp = restapi.incubator.min_temp
-    }
-
-    local body_json = sjson.encode(body_data)
-
-    return {
-        status = "200 OK",
-        type = "application/json",
-        body = body_json
-    }
-
-end -- end function
+function restapi.change_config(req)
+    -- Local variables
+    local request_body_json = req.getbody()
+    local body_table = sjson.decode(request_body_json)
+
+    -- Update the configuration values from the request body
+    if body_table.max_temperature then
+        body_table.max_temperature = tonumber(body_table.max_temperature)
```

Comment:
validar que sea un número 
intentar cambiar el valor en la incubadora 
```suggestion
        body_table.max_temperature = tonumber(body_table.max_temperature)
        restapi.incubator.set_max_temp(body_table.max_temperature)
```
el metodo set_max_temp, no existe pero deberia crearse para poner las guardas relacionadas con la logica de la incubadora en ese método. luego ese método cambiará el valor de la variable maxtemp.